### PR TITLE
fix document

### PR DESCRIPTION
**Pull request by superyyrrzz**
#1909

### CODE REVIEW

File: Documentation/tutorial/docfx.exe_user_manual.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ _disableBreadcrumb    | bool    | Indicate whether to show breadcrumb on the top
 _disableToc           | bool    | Indicate whether to show table of contents on the left of page.
 _disableAffix         | bool    | Indicate whether to show the affix bar on the right of page.
 _disableContribution  | bool    | Indicate whether to show the `View Source` and `Improve this Doc` buttons.
-_gitContribute        | object  | Customize the `Improve this Doc` URL button for public contributors. Use `repo` to specify the contribution repository URL. Use `branch` to specify the contribution branch. Use `path` to specify the folder for new overwrite files. If not set, the git URL and branch of the current git repository will be used.
+_gitContribute        | object  | Customize the `Improve this Doc` URL button for public contributors. Use `repo` to specify the contribution repository URL. Use `branch` to specify the contribution branch. Use `apiSpecFolder` to specify the folder for new overwrite files. If not set, the git URL and branch of the current git repository will be used.
```
Comment (hellosnow) [inline Documentation/tutorial/docfx.exe_user_manual.md]: >_gitContribute [](start = 0, length = 14)

should we provide an example of _gitContribute?

**Comment by hellosnow** [comment]
:shipit:

File: Documentation/tutorial/docfx.exe_user_manual.md (hunk 1 above)
Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: I think it may make this table cell too big...

Comment (MHDante) [inline Documentation/tutorial/docfx.exe_user_manual.md]: Please do so, the fields are ambiguous

Comment (superyyrrzz) [inline Documentation/tutorial/docfx.exe_user_manual.md]: Hi @MHDante, which description do you think ambiguous?

**Pull request merged**